### ISSUE TITLE
Verifiable Credential - KeyCloak integration POC - part 1

### ISSUE THREAD

**Issue by ShellyXueHan** [BCDevOps/developer-experience#565]
### Tasks:
Explore with VON team on how to integrate VC onto DEV KeyCloak admin account authentication:
- [x] Setup issuer and verifier
- [x] Setup IDP on keycloak
- [x] test out wallet app
- [x] test out issuing/invitation process
- [x] test out auth process
- [ ] test out revoking process

### Dependencies:
These are tasks outcome from a demo meeting, VON team will following up/work on the following.
- awaiting for credential revoking feature
- wallet app issues (Trinsic Wallet App):
   - mobile QR code scanning does not work
   - app quits from time to time
   - nice to have feature: automatic sovrin network recognition (so that users do not need to config settings)
 - wallet app update:
   - Since Trinsic Wallet does not work with revokable credentials, we need to switch to eSatus Wallet instead
   - app bug where request shows `unknow is requesting info for user`

**Comment by ShellyXueHan** [comment]
Update:
- Since Trinsic Wallet does not work wiuth revokable credentials, we need to switch to eSatus Wallet instead 
- the major parts are working fine with revocation feature, Emiliano just need to make a final change on it before we can test it out with internal users (PS team)

**Comment by esune** [comment]
[This](https://github.com/bcgov/issuer-kit/pull/227) should fix the workfllow issue: a new build is ongoing and should be ready for testing in the next few minutes.

Clarification: Trinsic is working on a fix for revocable credentials and a new version of the app should address this issue, however for the time being eSatus should be used for our testing.

**Comment by ShellyXueHan** [comment]
Not relavant.
